packages/rum-core/src/common/apm-server.js

```diff
@@ -207,25 +206,32 @@ class ApmServer {
   }
 
   ndjsonErrors(errors) {
-    return errors.map(function(error) {
-      return NDJSON.stringify({ error })
-    })
+    return errors.map(error => NDJSON.stringify({ error }))
+  }
+
+  ndjsonMetricsets(metricsets) {
+    const timestamp = Date.now() * 1000
```

Comment:
The timestamp should be captured by the Apm Server to avoid using wrong times on the clients as well as the issues with timezone.

Reply:
This would be blocker for breakdown graphs, but we can merge it behind feature flag.

Reply:
created an issue for the server - https://github.com/elastic/apm-server/issues/2700

Reply:
Found a way to fix this without getting affected by system clock drift. So its not a blocker anymore.